README.md

```diff
@@ -51,59 +46,111 @@ Example:
 ```yaml
 mysql_replication_user:
   name: replication
-  host: 192.168.25.1
+  host: 192.168.25.%
   password: "{{ vault_replication_password }}"
 ```
 
 ## Dependencies
 
 None.
 
-## Example Playbook
+## Example Variable & Playbook
+
+### Replication Primary-Primary
```

Comment:
Isn't this better described as “multi primary”? https://dev.mysql.com/doc/refman/8.0/en/group-replication-multi-primary-mode.html

Reply:
I didn't think of it! Done on the last version thanks :+1: